Data Structures/readme.md

```diff
@@ -1,4 +1,4 @@
-# Data Sctuctures
+# Data Structures
```

Comment:
Kindly try to some more info about Data Structures in the readme for changes to be approved.

Reply:
It was not for the hacktoberfest. I found this mistake and rectified it. Please remove the hacktoberfest tag from this. Cheers:) . It is not a contribution that is hacktoberfest worthy.

Reply:
I have added code for Insertion Sort. Now you can review it once and merge if you want. Cheers:)